src/Filter/TermFilter.php

```diff
@@ -2,22 +2,35 @@
 
 namespace Novaway\ElasticsearchClient\Filter;
 
+use Novaway\ElasticsearchClient\Query\CombiningFactor;
+
 class TermFilter implements Filter
 {
     /** @var string */
     private $property;
     /** @var string */
     private $value;
+    /** @var string */
+    private $combiningFactor;
 
     /**
      * TermFilter constructor.
      * @param string $property
      * @param string $value
      */
-    public function __construct(string $property, string $value)
+    public function __construct(string $property, string $value, string $combiningFactor = CombiningFactor::FILTER)
```

Comment:
The `$property`, `$value`, `$combiningFactor` seems to be commons in various class.

Maybe they could mutualized in a common parent ?